cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -64,6 +64,11 @@ public final class DataprocUtils {
   private static final int MIN_WAIT_TIME_MILLISECOND = 500;
   private static final int MAX_WAIT_TIME_MILLISECOND = 10000;
 
+  public static final String TROUBLESHOOTING_DOCS_URL_KEY =  "troubleshooting.docs.url";
```

Comment:
nit - generally we use camelcase, for eg `troubleshootingDocsURL`.

Reply:
Updated.